storm-dist/binary/src/main/assembly/binary.xml

```diff
@@ -340,10 +340,10 @@
         </fileSet>
         <!-- $STORM_HOME/extlib -->
         <fileSet>
-            <directory>${project.basedir}/../../external/storm-kafka-monitor/target</directory>
+            <directory></directory>
```

Comment:
I see. Since storm-kafka-monitor was being copied always, this section was not needed anymore. Thanks for clarifying.